src/main/drivers/serial_uart_impl.h

```diff
@@ -116,25 +125,33 @@
 
 typedef struct uartPinDef_s {
     ioTag_t pin;
-#if defined(STM32F7)
+#if defined(STM32F7) || defined(STM32H7)
     uint8_t af;
 #endif
 } uartPinDef_t;
 
 typedef struct uartHardware_s {
     UARTDevice_e device;    // XXX Not required for full allocation
     USART_TypeDef* reg;
+#ifdef USE_DMA
 #if defined(STM32F1) || defined(STM32F3)
     DMA_Channel_TypeDef *txDMAChannel;
     DMA_Channel_TypeDef *rxDMAChannel;
 #elif defined(STM32F4) || defined(STM32F7)
     uint32_t DMAChannel;
     DMA_Stream_TypeDef *txDMAStream;
     DMA_Stream_TypeDef *rxDMAStream;
+#elif defined(STM32H7)
+    // DMAMUX input from peripherals (DMA_REQUEST_xxx); RM0433 Table 110.
+    uint8_t txDMARequest;
```

Comment:
It will be slightly cleaner to keep ordering the same as in uartPort_s

Reply:
👍